retrieval/discovery/marathon/url.go

```diff
@@ -0,0 +1,14 @@
+package marathon
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+const appsURLFormat string = "http://%s/v2/apps/?embed=apps.tasks"
+
+// RandomAppsURL randomly selects a server from an array and creates an URL pointing to the app list
+func RandomAppsURL(servers []string) string {
```

Comment:
I seem to be missing the point, because I still do not get what TODO to add here. Could you explain the scenario you are thinking about? For me this sounds like people messing up their configuration and I am not sure how we can guard against that.